Library/Homebrew/dev-cmd/man.rb

```diff
@@ -18,6 +18,10 @@ def man
     else
       regenerate_man_pages
     end
+
+    git_diff_args = ["-C", HOMEBREW_REPOSITORY, "diff", "--stat"]
+    git_diff_args << "--exit-code" if ARGV.flag? "--exit-code"
+    exit system "git", *git_diff_args, "docs/brew.1.html", "manpages"
```

Comment:
Ok, since you don't want to display any `git diff`, I switched from `git diff --stat` to `git diff --quiet`, which implies `--exit-code`. I also renamed the `brew man --exit-code` option to `--fail-if-changed`, which is a bit more descriptive, and hopefully not too cumbersome.

Also, adding documentation of the `--fail-if-changed` option has made the manpages out of date in a bit of self-reference (updated in e748881).

The test-bot PR will be out of date if we stick with the `--fail-if-changed` option, since it's still using `--exit-code`.